src/agent/debuglet.ts

```diff
@@ -312,6 +312,7 @@ export class Debuglet extends EventEmitter {
         minorVersion_:
           process.env.GAE_DEPLOYMENT_ID || process.env.GAE_MINOR_VERSION,
       },
+      resetV8DebuggerThreshold: process.env.RESET_V8_DEBUGGER_THRESHOLD,
```

Comment:
It looks like you may need to:
```
rm -rf node_modules package-lock.json
npm install
npm run fix
```

We had a linter update that caused problems.

Reply:
Thanks for suggesting the fix. The new lint update turns out to affect lots files. I have manually included only the files that are affected by this changes.

Reply:
Uh, actually, I have to include all the files to solve the lint complaint. Will do it in the next commit.